ci/validate-formula.sh

```diff
@@ -0,0 +1,80 @@
+#! /bin/bash
+
+# this script is intended to be executed via PRs travis CI
+set -e
+
+# 01: hacert01
+
+echo "==========================================="
+echo " Using host hacert01                       "
+echo "==========================================="
+
+cp pillar.example ci/pillar/netweaver.sls
+cp ci/salt/top.sls .
+
+cat >grains <<EOF
+host: hacert01
+virtual_host: sapha1as
+sap_instance: ascs
```

Comment:
Is this entry in the `grains` file used for something? I don't think so, we could delete it in that case (the same for all entries)

Reply:
Hi @arbulu89 yes these entries are not needed. I remove these